.github/release-drafter.yml

```diff
@@ -0,0 +1,29 @@
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
```

Comment:
Should these be the conventional commit tags that we use? e.g. `feat`

Reply:
Yes